src/main/java/com/openmrs/migrator/core/services/BootstrapService.java

```diff
@@ -1,15 +1,11 @@
 package com.openmrs.migrator.core.services;
 
 import java.io.IOException;
-import java.nio.file.Path;
 import java.util.List;
-import java.util.Set;
-import java.util.stream.Stream;
 
 public interface BootstrapService {
 
-  Stream<String> createDirectoryStructure(List<String> dirList, Path settingsProperties)
-      throws IOException;
+  void createDirectoryStructure(List<String> dirList) throws IOException;
 
-  Set<String> populateDefaultResource(Set<String> sourceFiles) throws IOException;
+  void populateDefaultResources(List<String> sourceFiles) throws IOException;
 }
```

Comment:
hum! moving to void  return type we loose ability to test these methods. I  think  in the future we could have  more scenarios  needed to  be tested, so  return types instead if void  would be useful